packages/fxa-auth-server/.eslintrc.json

```diff
@@ -7,5 +7,11 @@
   "parserOptions": {
     "ecmaVersion": "2020",
     "sourceType": "module"
-  }
+  },
+  "root": true,
+  "ignorePatterns": [
+    "dist",
+    "fxa-*",
+    "vendor"
+  ]
```

Comment:
Looks fine.  But I'm a little confused why we didn't need to ignore `dist` before, and why we need to ignore `vendor`, which AFAICT doesn't exist in fxa-auth-server.

Reply:
I think we did need it before, but it wasn't there.
https://github.com/mozilla/fxa/blob/4aa5a6bf7906d88891afdefcc9e99a30d1873315/packages/fxa-auth-server/package.json#L12

The `build` command runs <kbd>tsc --build</kbd>, and the tsconfig.json file seems to have an `compilerOptions.outDir` of "./dist/", as well as the `build` command doing <kbd>cp -R config public lib scripts dist/fxa-auth-server</kbd>.
So I interpreted that as the ./dist/ folder being generated assets which didn't need to be [re]linted.

As for the mysterious "./vendor/" folder, it doesn't seem to be in the repo, but it _DOES_ seem to be a side effect of the `install-ejs` script in package.json; [`./scripts/install-ejs.sh`](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/scripts/install-ejs.sh), which seems to be fetching a GitHub mirror of ejs and copying it into a fresh copy of /vendor/ejs.js:

https://github.com/mozilla/fxa/blob/4aa5a6bf7906d88891afdefcc9e99a30d1873315/packages/fxa-auth-server/scripts/install-ejs.sh#L14-L15

Reply:
Gotcha.  Thanks for the explanation.